### PR TITLE
Update to use react-native-sentry 0.43.2

### PR DESCRIPTION
**Pull request by cruzach**
self-explanatory, tested locally and works smooooth

### CODE REVIEW

**Comment by cruzach** [comment]
@brentvatne after adding `expo-constants` as a dependency, do we still need to increase the major version number for the next release?

File: yarn.lock [hunk 1]
```diff
@@ -611,18 +618,30 @@ r2@^2.0.1:
...
+  version "3.27.2"
+  resolved "https://registry.yarnpkg.com/raven-js/-/raven-js-3.27.2.tgz#6c33df952026cd73820aa999122b7b7737a66775"
+  integrity sha512-mFWQcXnhRFEQe5HeFroPaEghlnqy7F5E2J3Fsab189ondqUzcjwSVi7el7F36cr6PvQYXoZ1P2F5CSF2/azeMQ==
+
+<<<<<<< Updated upstream
+react-native-sentry@^0.43.2:
+  version "0.43.2"
+  resolved "https://registry.yarnpkg.com/react-native-sentry/-/react-native-sentry-0.43.2.tgz#1505d16d334656f25f4d515e004d0c5a4ac5a01b"
+  integrity sha512-MDrMkt/ntyQhhn+b3EVUuB/lXvQbSLvCOpkAvioNOzddoyo2R6i3rVZ3aZmPN1oK3O9GFE4tSa0M+eTHni7AaQ==
+||||||| merged common ancestors
```
Comment (brentvatne) [inline yarn.lock]: merge conflicts

File: package.json [hunk 2]
```diff
@@ -6,8 +6,9 @@
   "license": "MIT",
   "dependencies": {
     "@expo/spawn-async": "^1.2.8",
+    "expo-constants": "^6.0.0",
```
Comment (brentvatne) [inline package.json]: we should make this `"expo-constants": "*"` so we don't have to update the package every release. this way we can just use whatever people have installed already. if the constants change in the future (they have been relatively stable) we'll need to update the library

**Comment by HazAT** [comment]
I was able to make `sentry-expo` work with our new react native SDK.
See: https://github.com/expo/sentry-expo/pull/58

I think we can close this or anyway not doing a major bump with this since ideally we only have not two breaking changes after another.

**Pull request closed without merging**